### DISCUSSION TITLE
can we expect a micromark 4.x update for mdast-util-from-markdown ?

### DISCUSSION THREAD

**Discussion by tripodsan** [syntax-tree/unist#103] [General] [answered]
Hi,
the current mdast-util-from-markdown still uses [micromark 3.x](https://github.com/syntax-tree/mdast-util-from-markdown/blob/e30d9c8025d726360123ae95beb9cf011335b921/package.json#L48), which causes problems when combining with other packages already dependent on 4.x. 

I would be helpful to know if an update is planned.
Thanks.

**Comment by wooorm** [comment, marked as answer]
Yes, updates are planned for the entire ecosystem. This takes time though. The problem with combining is only in the types, it otherwise works
